vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -48,6 +48,7 @@ type CodeLensOptions =
     UseColors: bool
     Prefix : string }
 
+[<CLIMutable>]
```

Comment:
Could you add a comment saying why its needed please, it won't be removed again? thanks